index.html

```diff
@@ -51,171 +52,155 @@
             </label>
           </div>
           <ul class="nav">
-            <li><a href="./pages/solution.html">solutions <i class="mobile fas fa-chevron-right"></i></a></li>
-            <li id="products"><a href="./pages/products.html">products <i class="mobile fas fa-chevron-right"></i></a>
+            <li>
+              <a href="./pages/solution.html">solutions</a>
+              <label class="mobile">
+                <i class="fas fa-chevron-right"></i>
+              </label>
+            </li>
+            <li id="products">
+              <a href="./pages/products.html">products</a>
+              <label for="sup-menu-check" class="mobile">
+                <i class="fas fa-chevron-right"></i>
+              </label>
+              <input type="checkbox" name="sup-menu-check" id="sup-menu-check" class="hidden-input" aria-hidden="true">
               <div class="menu-items hide-mobile">
                 <div class="container">
+                  <label class="sup-menu-header mobile" for="sup-menu-check"><i class="fa-solid fa-arrow-left"></i>
+                    products</label>
                   <div class="menu-items-lvl1 sup-menu">
                     <ul>
+
                       <li id="boilers">
+                        <input type="checkbox" name="sup-menu-lvl2-check" id="sup-menu-lvl2-check" class="hidden-input"
+                          aria-hidden="true">
                         <a href="#">
```

Comment:
fix the url
if it's not direct to anywhere try javascript:void(0)

Reply:
ok great thanks

Reply:
Done